istio-envoy-1.18.yaml

```diff
@@ -1,8 +1,8 @@
 package:
   name: istio-envoy-1.18
   # On the next version bump, please remove the make 4.4 patch.
```

Comment:
Do we need to remove the patch below?

Reply:
I removed now. If Istio bumped their Envoy deps it will work, if they cherrypicked it wouldn't. 🤞